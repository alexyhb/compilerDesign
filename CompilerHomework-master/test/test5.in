normal|split;here and spaces
not\;here\|though
split\ me\ not
finishing\;;then\\;
