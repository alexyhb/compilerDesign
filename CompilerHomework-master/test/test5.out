Text ->normal<-
Pipe
Text ->split<-
Semi
Text ->here<-
Blank 1 chars
Text ->and<-
Blank 1 chars
Text ->spaces<-
NL
Text ->not\;here\|though<-
NL
Text ->split\ me\ not<-
NL
Text ->finishing\;<-
Semi
Text ->then\\<-
Semi
NL
